### PR TITLE
feat: Add limit and search flags to stack list command

### PR DESCRIPTION
**Pull request by 0michalsokolowski0**
Adds limit and search flags to stack list command

### CODE REVIEW

File: internal/helpers.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package internal
+
+func ToPtr[T any](v T) *T {
```
Comment (mbialon) [inline internal/helpers.go:3]: Nit: `Ptr` would be fine.

**Review by mbialon — APPROVED** [review]
LGTM

File: internal/cmd/stack/list.go [hunk 2]
```diff
@@ -18,20 +21,57 @@ func listStacks() cli.ActionFunc {
 			return err
 		}
 
+		var limit *uint
+		if cliCtx.IsSet(flagLimit.Name) {
+			if cliCtx.Uint(flagLimit.Name) == 0 {
+				return fmt.Errorf("limit must be greater than 0")
+			}
+
+			if cliCtx.Uint(flagLimit.Name) >= math.MaxInt32 {
```
Comment (tomasmik) [inline internal/cmd/stack/list.go:30]: Are such checks really necessary? No point in removing it now, but also eeh 🤷🏻

Comment (Yantrio) [reply internal/cmd/stack/list.go:30]: I like it :) Validation is helpful

File: internal/helpers.go (hunk 1 above)
Comment (0michalsokolowski0) [reply internal/helpers.go]: 8d92c92027f44e5d3683a7d8d837dbc0cc29ebf9

File: internal/cmd/stack/list.go (hunk 2 above)
Comment (0michalsokolowski0) [reply internal/cmd/stack/list.go:30]: It might be a matter of taste, but I like knowing what to expect from inputs  :)

**Pull request merged**